k8s/sample-pv-yamls/pvc-jiva-sc-beta-1r.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: jiva-beta-sc
+  annotations:
+    openebs.io/cas-type: jiva
+    cas.openebs.io/config: |
+      - name: ReplicaCount
+        value: "1"
+provisioner: openebs.io/provisioner-iscsi
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: jiva-beta-sc-claim
+  volume.beta.kubernetes.io/storage-class: jiva-beta-sc
```

Comment:
Also add this line: volume.beta.kubernetes.io/storage-provisioner: openebs.io/provisioner-iscsi